user_docs/en/userGuide.md

```diff
@@ -2475,8 +2476,8 @@ This volume can also be controlled via the following keyboard commands from anyw
 
 | Name | Key | Description |
 |---|---|---|
-| Increase application volume | `NVDA+alt+pageUp` | Increases volume of all applications except NVDA. |
-| Decrease application volume | `NVDA+alt+pageDown` | Decreases volume of all applications except NVDA. |
+| Increase application volume | `NVDA+alt+pageUp` | Increases the volume of all applications except NVDA. |
+| Decrease application volume | `NVDA+alt+pageDown` | Decreases the volume of all applications except NVDA. |
```

Comment:
```suggestion
| Increase applications volume | `NVDA+alt+pageUp` | Increases the volume of all applications except NVDA. |
| Decrease applications volume | `NVDA+alt+pageDown` | Decreases the volume of all applications except NVDA. |
```

Reply:
"applications volume" doesn't sound right to me. How about increase/decrease volume of other applications?

Reply:
That would be fine too, I just think the current writing implies it only changes the current applications volume